src/Components/Scenes/SearchResults/SearchResults/SearchResults.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/styles';
+import Splash from '../../../Presentational/Splash';
+import TableSectionContainer from '../../SearchResults/TableSection/TableSection-Container';
+import TableSkeleton from '../../SearchResults/TableSkeleton/TableSkeleton';
+
+const searchResultsStyle = makeStyles(theme => ({
+  // not certain if we will want or need separate styles at the moment
+  searchResults: {}

```

Comment:
If we are not using it we can remove it.

Reply:
thank you. will remove, kind of forgot to address that haha